### ISSUE TITLE
Unsaved R script files have an * appended to file name - can this be enhanced with color?

### ISSUE THREAD

**Issue by TidyStewart** [rstudio/rstudio#11038]
When editing scripts in R Studio's 'source' pane / panel, the unsaved file name has a * appended to it (in the tab) to remind you that the work needs saved.  Is it possible to also change the *, or even the whole file name, to red? 

This would make it easier to remind me to save my edits. Is there a setting for this? Thanks

**Comment by romainfrancois** [comment]
Which version are you using. This appears to work with recent daily builds. 

<img width="312" alt="image" src="https://user-images.githubusercontent.com/2625526/164707434-3cfddb5a-617a-429d-9a63-efdee72661b3.png">

**Comment by TidyStewart** [comment]
I am using 2022.02.0 Build 443

**Comment by TidyStewart** [comment]
Sorry!!!!!!!  It was the Theme I was using.  It seems like if you use a 'darker' theme, the red text does not work.

**Comment by kevinushey** [comment]
Indeed, for dark themes, the text should be slightly darker rather than red.

<img width="209" alt="Screen Shot 2022-04-22 at 9 23 50 AM" src="https://user-
images.githubusercontent.com/1976582/164755343-fcac36d6-c1c6-4188-953a-2712791b2505.png">
